src/dataExplorer/components/ResultsPane.tsx

```diff
@@ -58,37 +56,10 @@ const ResultsPane: FC = () => {
   }
 
   const submit = () => {
-    let timeRangeStart, timeRangeStop
-
-    if (!timeRange) {
-      timeRangeStart = timeRangeStop = null
-    } else {
-      if (timeRange.type === 'selectable-duration') {
-        timeRangeStart = '-' + timeRange.duration
-      } else if (timeRange.type === 'duration') {
-        timeRangeStart = '-' + timeRange.lower
-      } else if (isNaN(Date.parse(timeRange.lower))) {
-        timeRangeStart = null
-      } else {
-        timeRangeStart = new Date(timeRange.lower).toISOString()
-      }
-
-      if (!timeRange.upper) {
-        timeRangeStop = 'now()'
-      } else if (isNaN(Date.parse(timeRange.upper))) {
-        timeRangeStop = null
-      } else {
-        timeRangeStop = new Date(timeRange.upper).toISOString()
-      }
-    }
-
     setStatus(RemoteDataState.Loading)
     const time = Date.now()
     query(text, {
-      vars: {
-        timeRangeStart,
-        timeRangeStop,
-      },
+      variables: !!timeRange ? getTimeRangeVars(timeRange) : [],
```

Comment:
instead of using `timeRangeStart` and `timeRangeStop` as on override (or an override)...just pass the timeRange Variables.

Reply:
more of the same. this is a code smell -- but leaving ResultsPane and notebooks alone for now.